content-src/lib/selectLayoutRender.js

```diff
@@ -37,7 +37,7 @@ export const selectLayoutRender = createSelector(
         }
 
         // Calculate if we should display a spoc or not.
-        if (component.spocs) {
+        if (component.spocs && spocs.data && spocs.data.spocs && spocs.data.spocs.length) {
```

Comment:
I *think* we can remove the `spocs.data` part in the checks as that comes from reduxand *should* always be defaulted.

Ending up with just `if (component.spocs && spocs.data.spocs && spocs.data.spocs.length) {`